### ISSUE TITLE
For some reason in Discord chats from in-game appear with <> around every letter

### ISSUE THREAD

**Issue by DoubleZPlaying** [arthurbambou/Fabric---Discord-Link#141]
For some reason in Discord chats from in-game appear with <> around every letter. If I type Hello into game chat it appears as <H><e><l><l><o>

How do I fix this?

 

Here's my code:

 

{
"minecraftToDiscord": {
"playerMessage": {
"customMessage": "\u003c%player\u003e %message",
"useCustomMessage": true
},
"teamPlayerMessage": {
"customMessage": "%team \u003c%player\u003e %message",
"useCustomMessage": true
},
"serverStarting": "Server is starting!",
"serverStarted": "Server Started.",
"serverStopping": "Server is stopping!",
"serverStopped": "Server Stopped.",
"channelDescription": "Playercount : %playercount/%maxplayercount,\n Uptime : %uptime",
"playerJoined": {
"customMessage": "%player joined the game",
"useCustomMessage": true
},
"playerJoinedRenamed": {
"customMessage": "%new (formerly known as %old) joined the game",
"useCustomMessage": true
},
"playerLeft": {
"customMessage": "%player left the game",
"useCustomMessage": true
},
"advancementTask": {
"customMessage": "%player has made the advancement %advancement",
"useCustomMessage": true
},
"advancementChallenge": {
"customMessage": "%player has completed the challenge %advancement",
"useCustomMessage": true
},
"advancementGoal": {
"customMessage": "%player has reached the goal %advancement",
"useCustomMessage": true
},
"deathMsgPrefix": "",
"deathMsgPostfix": "",
"atATellRaw": "%message",
"meMessage": {
"customMessage": "* %author %message",
"useCustomMessage": true
},
"adminMessage": {
"customMessage": "[%author: %message]",
"useCustomMessage": true
},
"sayMessage": {
"customMessage": "[%author] %message",
"useCustomMessage": true
},
"achievement": {
"customMessage": "%player has just earned the achievement %achievement",
"useCustomMessage": true
}
},
"discordToMinecraft": {
"message": "[%player] %message"
},
"discord": {
"commandPrefix": "!",
"botActivities": [
"!commands",
"%playercount / %maxplayercount",
"on %ip",
"%uptime_D day(s), %uptime_H hour(s), %uptime_M minute(s) and %uptime_S second(s)"
]
}
}

**Comment by thecatcore** [comment]
do you have any other mod installed?

**Comment by DoubleZPlaying** [comment]
> do you have any other mod installed?

Yes. None of them should affect chat though. Here's the list:
Inv View
Deltalogger
Fabric API
Image2Map
Lithium
Sit
Spark
Survival Spectator

The version is 1.16.5

**Comment by thecatcore** [comment]
I can't reproduce the issue rn, could you send the other config file too?

**Comment by DoubleZPlaying** [comment]
[fdlink.zip](https://github.com/arthurbambou/Fabric---Discord-Link/files/6288227/fdlink.zip)
This is the fdlink folder in my config folder

> I can't reproduce the issue rn, could you send the other config file too?

**Comment by thecatcore** [comment]
I'm looking at it rn

**Comment by DoubleZPlaying** [comment]
Perfect

**Comment by arthurbambou** [comment]
can you try after removing the empty emoji in emoji map?
```
"emojiMap": [
    {
      "name": "chat",
      "id": "829112994873081857"
    },
    {
      "name": "",
      "id": ""
    }
  ]
```

the empty one is this one:
```
{
      "name": "",
      "id": ""
}
```

**Comment by thecatcore** [comment]
It will be fixed in 0.8.18 when it will be released in a few minutes.

**Comment by DoubleZPlaying** [comment]
That fixed it! Thanks!
